schema/project-level/codelists/climateMeasures.csv

```diff
@@ -0,0 +1,10 @@
+Code,Title,Description,Source
+embodiedEnergyReduction,Reduction of embodied energy,"Measures that reduce the sum of energy required to produce, transport and repair or replace the materials used by the project.",OC4IDS
+embodiedEmissionReduction,Reduction of embodied emissions,"Measures that reduce the sum of emissions generated by the production, transportation and replacements or repair of materials used by the project.",OC4IDS
+fugitiveEmissionReduction,Reduction of fugitive emissions,"Measures that reduce greenhouse gas emissions and air pollutants from intentional or unintentional releases, such as methane during transport of natural gas or HFC emission from refrigeration equipment.",OC4IDS
+renewableEnergySources,Use of renewable energy sources,The project adopts renewable energy sources and minimizes the use of fossil fuels as an energy source.,OC4IDS
+carbonCapture,Use of carbon capture technology,The project incorporates carbon capture and storage technologies.,OC4IDS
+recycledMaterials,Use of recycled and/or reused materials,"Measures that reduce the use of virgin natural resources and avoid sending useful materials to landfills by specifying reused materials, including structures, and material with recycled content.",OC4IDS
+regenerativeDesign,Regenerative design,"The project has used a regenerative design approach. Regenerative design is a system of technologies and strategies, based on an understanding of the inner working of ecosystems that generates designs to regenerate rather than deplete underlying life support systems and resources within socio-ecological wholes.",OC4IDS
```

Comment:
Simplify by focusing on the words that make this concept distinct.

```suggestion
regenerativeDesign,Regenerative design,"The project has used a regenerative design approach. A regenerative design approach aims to regenerate, rather than deplete, life support systems and resources within socio-ecological systems.",OC4IDS
```